MsmqReturnToSourceQueue/MsmqUtilities.cs

```diff
@@ -7,15 +7,23 @@
 using System.Net.NetworkInformation;
 using System.Text;
 using System.Xml;
+using System.Xml.Linq;
 
-class MsmqUtilities
+public class MsmqUtilities
```

Comment:
super minor but in the context of `MsMqAddress` i'd say this should be `internal`

Reply:
Tests